x-pack/plugins/cases/public/components/create/flyout/use_cases_add_to_new_case_flyout.test.tsx

```diff
@@ -59,6 +60,45 @@ describe('use cases add to new case flyout hook', () => {
     expect(dispatch).toHaveBeenCalledWith(
       expect.objectContaining({
         type: CasesContextStoreActionsList.OPEN_CREATE_CASE_FLYOUT,
+        payload: expect.objectContaining({
+          attachments: undefined,
+        }),
+      })
+    );
+  });
+
+  it('should dispatch the open action when invoked with attachments', () => {
+    const { result } = renderHook(
+      () => {
+        return useCasesAddToNewCaseFlyout({});
+      },
+      { wrapper }
+    );
+    result.current.open({ attachments: [alertComment] });
+    expect(dispatch).toHaveBeenCalledWith(
+      expect.objectContaining({
+        type: CasesContextStoreActionsList.OPEN_CREATE_CASE_FLYOUT,
+        payload: expect.objectContaining({
+          attachments: [alertComment],
+        }),
+      })
+    );
+  });
+
+  it('should dispatch the open action when invoked, with custom attachments', () => {
```

Comment:
Probably I don't see it 🙂 but what is the difference between the two tests? Where are the custom attachments?

Reply:
This is a left over test! I'll remove it.